internal/pkg/types/atto_fil.go

```diff
@@ -76,11 +39,11 @@ func (z *AttoFIL) UnmarshalBinary(b []byte) error {
 // Reasons for embedding a big.Int instead of *big.Int:
 //   - We don't have check for nil in every method that does calculations.
 //   - Serialization "symmetry" when serializing AttoFIL{}.
-type AttoFIL struct{ val big.Int }
+type AttoFIL = specsabi.TokenAmount
```

Comment:
I would rather purge the type completely. Why is it still here?

Reply:
Id also like to use one thing but if we want to remove it completely and do it the right way I'm going to need to make PRs to specsactors (attofil from fil, string manipulation) that this will need to block on.  I figured this intermediate state was better than nothing and didn't want to get involved in a longer multi-project dependency waiting game (particularly because [I'm already in one of those](https://github.com/ipfs/go-hamt-ipld/pull/43).

Another thing: this name is significantly better than TokenAmount and since we need to leave functionality here I figured we could also leave the better name.